### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by ignatius22**
### Made BooksForm component stateful.
     - stored components title and the book category in its own state.
     - Implemented the handleChange method which reacts to the user typing to input a name or to changes in 
         the category select.
    - Made form to modify the component’s state.
    - Implemented handleSubmit to  save the new book in the Redux store and resets the component’s 
      state.

### Connected the REMOVE_BOOK action to the component.
       - Add a new table column to the BooksList component. It should render the “Remove Book” button next to each book.
       - Implemented the handleRemoveBook method which accepts a book as an argument and removes it from the Redux store. 
        Don’t forget to pass that method down to the Book component as a prop.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**Good work so far.**

### STATUS: CHANGES REQUESTED 💢

- [x] Kindly check the comments in your code and make the necessary changes.

Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/containers/BookForm.js [hunk 1]
```diff
@@ -1,19 +1,79 @@
...
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  async handleChange(e) {
+    const { name } = e.target;
+    this.setState({
+      [name]: e.target.value,
+    });
+  }
```
Comment (mikenath223) [inline src/containers/BookForm.js]: - [x] OPTIONAL - No use making this function async. Consider making it a regular function. Async usually goes with await and so far in this there is n o async event to wait for a return value from it.

**Comment by mikenath223** [comment]
- [x] Kindly add a Remove book button to the Book component. And also attach the handleRemoveBook function to the said button.

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: 
Hi team :hand: 
Great job implementing the requested changes. Since all requested changes were made, I'm approving this PR.

- However, there is an issue in your application right now that allows books to be created without a name. Please kindly fix this issue since this project will have great value in your portfolio when looking for jobs.

Happy Coding :computer: 
**Sergio Torres**

**Pull request merged**